CONTRIBUTING.adoc

```diff
@@ -0,0 +1,57 @@
+= Contributing
+
+In the spirit of free software, _everyone_ is encouraged to help improve this project.
+
+To contribute code, simply fork the project on GitHub, hack away and send a pull request with your proposed changes.
+
+Feel free to use the https://github.com/asciidoctor/atom-language-asciidoc/issues[issue tracker] to provide feedback or suggestions in other ways.
+
+== Development
+*todo*
```

Comment:
Its personal but I think separate title from section content with a blank line is more readable.